doc/dev.html

```diff
@@ -64,8 +64,26 @@
 
 <hr />
 
-<h1 id="tasks">Compiling, testing, releasing, etc.</h1>
+<h1 id="setup">Setting up</h1>
+
+Randoop development involves working with both Randoop's code base and the
+website and documentation.
+The first is is in the master (and other branches) of the repository, while
+the website is in the gh-pages branch.
+These instructions assume that you keep subdirectories for both your working
+branch, and the gh-pages branch of Randoop in the same directory.
+Assuming this directory is <code>randoop-working</code>, get started by these steps:
+<code><pre>
+  cd randoop-working
+  git clone git@github.com:randoop/randoop.git myfeature
+  git clone git@github.com:randoop/randoop.git randoop-gh-pages
+  cd randoop-gh-pages
+  git checkout gh-pages
+</pre></code>
+You would then work in the directory <code>myfeature</code>, and follow the
+directions below to move changes into <code>gh-pages</code>.
```

Comment:
I removed this. I didn't understand that you were doing a checkout when I wrote it.

But I do think something is needed to orient the developer to how things work.   It is not unusual for dev documentation to give the clone command.  It would be appropriate to give the clone command and then explain that make distribution-files will dump a gh-pages directory into the parent.  It should also be explained that the command-line argument documentation comes from the code.